addons/mail/models/mail_tracking_value.py

```diff
@@ -39,7 +39,9 @@ def _compute_field_groups(self):
         for tracking in self:
             model = self.env[tracking.mail_message_id.model]
             field = model._fields.get(tracking.field.name)
-            tracking.field_groups = field.groups
+            # The field could not exists anymore, for example when
+            # calling message_change_thread
+            tracking.field_groups = field.groups if field else 'base.group_system'
```

Comment:
@dbeguin Not sure what to do with this fix.

Does this apply to prior stable versions?
Could you add a unit test to demonstrate the issue and also ensure this fixes it?

Thanks :)

Reply:
+1 a test would be welcomed :)

Reply:
Well that was already fixed but the fix was stuck in 13.0.
I took here the 'manual FP recovering' commit, that have been made in 13.2, to make the ENT branch working while testing the feature.